iot-hub/Samples/service/RoleBasedAuthenticationSample/Parameters.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.Azure.Devices;
+
+namespace RoleBasedAuthenticationSample
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'h',
+            "HostName",
+            Required = true,
+            HelpText = "The IoT hub host name.")]
+
+        public string HostName { get; set; }
+
+        [Option(
+           'd',
+           "DeviceId",
+           Required = true,
+           HelpText = "The IoT hub device to send a message to.")]
+
+        public string DeviceId { get; set; }
+
+        [Option(
+            't',
+            "TransportType",
+            Default = TransportType.Amqp,
```

Comment:
we've been trying to get customers to default to mqtt instead of amqp (with mqtt becoming a bigger focus these days), we should probably do the same here as well